website/source/docs/drivers/raw_exec.html.md

```diff
@@ -35,6 +36,9 @@ options = {
 }
 ```
 
+You must specify either a `command` or a `artifact_source` to be executed. Any
```

Comment:
This should reflect the updated design. E.g. "must specify command, may specify artifact_source to download"